export.go

```diff
@@ -50,12 +50,12 @@ func newExporter(tree *ImmutableTree) *Exporter {
 
 // export exports nodes
 func (e *Exporter) export(ctx context.Context) {
-	e.tree.root.traversePost(e.tree, true, func(node *Node) bool {
+	e.tree.root.TraversePost(e.tree, true, func(node ComplexNode) bool {
 		exportNode := &ExportNode{
```

Comment:
Should export `ExportNode` implement `Node` interface?

Reply:
Exporter exports nodes from an ImmutableTree which is made up of ComplexNode not Node. as we need the Height() function